protocol/messenger_store_node_request_manager.go

```diff
@@ -185,7 +252,9 @@ type storeNodeRequest struct {
 
 	// request corresponding metadata to be used in finalize
 	filterToForget *transport.Filter
-	communityID    string
+
+	requestType storeNodeRequestType
+	typedID     string // FIXME: rename
```

Comment:
fixme - is it ok?

Reply:
no, it's not 😄 
Will fix now, thank you!